charts/lily/values.yaml

```diff
@@ -139,9 +136,9 @@ debug:
       cpu: "8000m"
       memory: "16Gi"
 
-logLevel: info
-logLevelNamed: "vm:error,badgerbs:error"
 logFormat: json
+logLevel: warn
```

Comment:
I'd still like some info logs from lily, else its operation will be fairly opaque. What if we set all non-lily-logs to error and left the lily loggers at level `info`?

Reply:
Yeah I'd like it as the default.